src/Moryx.AbstractionLayer/Capabilities/Capabilities.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) 2021, Phoenix Contact GmbH & Co. KG
+// Licensed under the Apache License, Version 2.0
+
+namespace Moryx.AbstractionLayer.Capabilities
+{
+    /// <summary>
+    /// Preferred base class for implementations of <see cref="ICapabilities"/>
+    /// </summary>
+    public abstract class Capabilities : ConcreteCapabilities
```

Comment:
Could you suppress the compiler warning? There are some other, which you can see in the build output or on you local visual studio.

Reply:
Done